src/bidiMapper/CommandProcessor.ts

```diff
@@ -78,17 +82,18 @@ export class CommandProcessor extends EventEmitter<CommandProcessorEvents> {
       preloadScriptStorage,
       logger
     );
+    this.#cdpProcessor = new CdpProcessor(
+      browsingContextStorage,
+      cdpConnection
+    );
     this.#inputProcessor = InputProcessor.create(browsingContextStorage);
```

Comment:
Nit: Can we refactor away the .create we don't do anything async so no need for it, we should just put all things in the constructor